src/MeshKernelNET/Native/MeshKernelDll.cs

```diff
@@ -124,6 +124,16 @@ static MeshKernelDll()
                                                                                [In] ref CurvilinearParametersNative curvilinearParametersNative,
                                                                                [In] ref SplinesToCurvilinearParametersNative splinesToCurvilinearParametersNative);
 
+        /// <summary>
+        /// Computes the smoothness of a curvilinear grid.
+        /// </summary>
+        /// <param name="meshKernelId">Id of the mesh state</param>
+        /// <param name="direction">The direction in which to compute the smoothness (0 for m direction, 1 for n direction)</param>
+        /// <param name="smoothness">The grid smoothness values in the selected direction</param>
+        /// <returns>Error code</returns>
```

Comment:
This is an exit code really which may be success (0) or error ( > 0). This doc error is everywhere, not only here. I'd leave it for now.